GDJS/Runtime/runtimescene.ts

```diff
@@ -86,8 +90,7 @@ namespace gdjs {
         : 0;
       for (const name in this._layers.items) {
         if (this._layers.items.hasOwnProperty(name)) {
-          /** @type gdjs.Layer */
-          const theLayer: gdjs.Layer = this._layers.items[name];
+          const theLayer: gdjs.RuntimeLayer = this._layers.items[name];
```

Comment:
Is `/** @type gdjs.Layer */` useful in some way or is it a remanent comment from a long forgotten JS past?

Reply:
Remanent from a long forgotten JS past 👻